src/rez/pip.py

```diff
@@ -102,6 +102,9 @@ def get_distribution_name(pip_name):
                 version = version.replace("==", "")
                 name = get_distribution_name(name)
             except DistlibException:
+                # check if package contains erroneous additional environment info and remove it
```

Comment:
erroneous? Do you meant extraneous?

I would add a link to the distlib docs on markers to clarify what's happening here.